src/PresentationalComponents/Modals/DisableRule.js

```diff
@@ -94,6 +93,12 @@ const DisableRule = ({ handleModalToggle, intl, isModalOpen, host, hosts, rule,
                     aria-describedby="disable-rule-justification"
                     value={justification}
                     onChange={(text) => setJustificaton(text)}
+                    onKeyDown={(e) => {
```

Comment:
ca
```suggestion
                    onKeyDown={e => e.key === 'Enter' && (e.preventDefault(); disableRule();)}
```

does that still work?